diff --git a/node_modules/@react-native-tvos/config-tv/build/withTVSplashScreen.js b/node_modules/@react-native-tvos/config-tv/build/withTVSplashScreen.js
index f7edfe3..e331267 100644
--- a/node_modules/@react-native-tvos/config-tv/build/withTVSplashScreen.js
+++ b/node_modules/@react-native-tvos/config-tv/build/withTVSplashScreen.js
@@ -26,6 +26,15 @@ const withTVSplashScreen = (config, params = {}) => {
                 property: 'splashscreen',
             });
             await fs_1.promises.writeFile(file, modifiedContents, 'utf-8');
+            const devLauncherFile = path_1.default.join(config.modRequest.projectRoot, 'node_modules', 'expo-dev-launcher', 'ios', 'Views', 'EXDevLauncherErrorView.storyboard');
+            const devLauncherContents = await fs_1.promises.readFile(devLauncherFile, 'utf8');
+            const modifiedDevLauncherContents = addTVSplashScreenModifications(devLauncherContents);
+            (0, utils_1.verboseLog)('modifying dev launcher storyboard for tvOS', {
+                params,
+                platform: 'ios',
+                property: 'splashscreen'
+            });
+            await fs_1.promises.writeFile(devLauncherFile, modifiedDevLauncherContents, 'utf-8');
             return config;
         },
     ]);
